### PR TITLE
docs(events): adds documentation to events

### LINKED ISSUE #603: documentation: Add documentation about events (render)
I didn't find any doc about events? do we have any to hook some functions?

### PR DESCRIPTION
**Pull request by bobylito**
Fix #603

### CODE REVIEW

File: docs/documentation.md [hunk 1]
```diff
@@ -265,6 +265,39 @@ This example shows you how to create a very simple search results page that incl
...
+// until removeListener is called
+search.removeListener(onRenderHandler);
+
+search.once('render', function(){  });
+// triggered once then removed automatically
+{% endhighlight %}
+  </div>
+</div>
+
+InstantSearch emits event during its lifecylce. The currently supported events are:
```
Comment (vvo) [inline docs/documentation.md]: it's instantsearch :D

File: docs/documentation.md [hunk 2]
```diff
@@ -265,6 +265,39 @@ This example shows you how to create a very simple search results page that incl
...
+
+ - `render` : fired when a rendering of all the widgets has been completed
+
+InstantSearch events are based on the [Node EventEmitter](https://nodejs.org/api/events.html)
+class. See the example on the right for a quick overview of the API or go to the
```
Comment (vvo) [inline docs/documentation.md]: Careful because "see the example on the right" only works at certain screen size

File: docs/documentation.md (hunk 1 above)
Comment (bobylito) [inline docs/documentation.md]: Noted!

File: docs/documentation.md (hunk 2 above)
Comment (bobylito) [inline docs/documentation.md]: Fixed :)

**Comment by bobylito** [comment]
Looks like we're good here :) Merging!

**Pull request merged**